cluster/cli.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
```

Comment:
I really don't want for us to expand into another top-level directory. Can we put this in `hack/`?

Reply:
Same question for everything else in `cluster/`.

Reply:
This is because those scripts are placed also in the `cluster/` directory on the other CNV projects (kubevirt, CNA, CDI ....) and I think it's important the directory tree will be the same (more or less) between all of these projects, so it will be easier for others to contribute.